src/lsptoolshost/extensions/roslynLanguageServerExportChannel.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import * as vscode from 'vscode';
-import { RequestType } from 'vscode-languageclient/node';
+import { RequestType } from 'vscode-jsonrpc';
```

Comment:
What happened here - did types get moved around?  Would have thought we would be using the language client API